graylog2-web-interface/src/components/datanode/migrations/remoteReindexing/MigrateExistingData.tsx

```diff
@@ -108,15 +109,25 @@ const MigrateExistingData = ({ currentStep, onTriggerStep }: MigrationStepCompon
   };
 
   return (
-    <Formik initialValues={initialValues} onSubmit={() => {}}>
+    <Formik initialValues={initialValues}
+            onSubmit={() => {
+            }}>
       {({
         values,
         setFieldValue,
       }) => (
         <Form role="form">
+          <Input id="allowlist"
```

Comment:
we need to provide more info in this form to make it easier to understand, e.g. example values